src/views/Wizard/views/Mesh.spec.ts

```diff
@@ -0,0 +1,72 @@
+import userEvent from '@testing-library/user-event'
+import { screen } from '@testing-library/vue'
+import renderWithVuex from '@/testUtils/renderWithVuex'
+import Mesh from './Mesh.vue'
+
+describe('Mesh.vue', () => {
+  const doStep = async (button: any, target: string | string[]) => {
+    // Begin step
+
+    expect(button).not.toHaveAttribute('disabled')
+
+    // Check extended validation
+
+    await userEvent.click(screen.getByText('Enabled'))
+
+    expect(button).toHaveAttribute('disabled')
+
+    if (typeof target === 'string') {
+      await userEvent.type(screen.getByLabelText(target), 'fake-name')
+    } else {
+      const map = target.map(targetName =>
+        userEvent.type(screen.getByLabelText(targetName), 'fake-name'),
+      )
+
+      await Promise.all(map)
+    }
+
+    expect(button).not.toHaveAttribute('disabled')
+
+    await userEvent.click(screen.getByText('Next ›'))
+  }
+
+  it('passes whole wizzard and render yaml', async () => {
+    const {
+      container,
+    } = renderWithVuex(Mesh, {
+      store: {
+
```

Comment:
Nit: unnecessary empty line

Reply:
Nice catch. I think I will provide husky and pre-commit hook that will take care of these kinds of "linting" issues. For now, I will skip it.